src/AuditTrailModule/javasource/audittrail/log/CreateLogObject.java

```diff
@@ -87,11 +96,17 @@ private static synchronized void setAssociationName(String otherObjectType, Stri
 		associationMapping.put(otherObjectType, associationName);
 	}
 
+	private static void incNumberOfChangedMembers(IMendixObject logObject, IContext sudoContext,
+			IContext currentContext, boolean isNew, String memberName) {
+		if (isNew) _logNode.trace("Member: " + memberName + " was added.");
+		else _logNode.trace("Member: " + memberName + " has changed.");
+		
+		logObject.setValue(sudoContext, Log.MemberNames.NumberOfChangedMembers.toString(),
+				(Integer) logObject.getValue(currentContext, Log.MemberNames.NumberOfChangedMembers.toString()) + 1);
+	}
+
 	public static IMendixObject CreateAuditLogItems(IMendixObject inputObject, IContext context) throws CoreException {

```

Comment:
While we are at it let's use the lowerCamelCase in the method's name.

Reply:
Do you think there's a risk to existing users if we do? I mean, it will fail to compile otherwise and any external use of this method will be easily flagged, so no problem there, correct?